_posts/2022-01-14-code-reviews.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Reviewing finished code is too late - and other tips and motivations for code reviews
```

Comment:
Changed the title according to feedback. Tried to put one thing from the blog post to here ("Reviewing finished code is too late"). But as it is not the only thing in the blog post, put general part after - ("and other tips and motivations for code reviews").

Reply:
On the other hand, code is "never finished" :) Maybe something like: There is never a perfect time to review your code: do it!

Reply:
I kind of liked you're first proposal `Code review is more important than progamming`

Reply:
Sorry for late reply. I also liked my original "code review is more important than programming" even if I wasn't sure about it. So I changed to it. Many good options for the title.